sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -187,19 +193,21 @@ $(document).on('click', '.postCourseNameChange', function (e) {
   e.preventDefault();
   var thisCourse = $(this).attr('data-courseid');
   var newCourseName = $(this).closest('.courseTitleTextContainer').find('input.courseTitleText').val();
-  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;;
+  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;
   var $thisCourseCode = $(this).closest('li').find('.courseLink');
   var $thisCourseName = $(this).closest('li').find('.courseName');
+  var position = e.pageX;
+
   $.ajax({
     type: 'PUT',
     url: url
     }).done(function( msg ) {
      $('.courseTitleTextContainer').hide();
-      reportSuccess('Course <strong>' + $thisCourseCode.text() + '</strong> renamed to <strong>' + msg.course_code) + '</strong>';
+      reportSuccess(position, 'Course <strong>' + $thisCourseCode.text() + '</strong> renamed to <strong>' + msg.course_code + '</strong>');
       $thisCourseCode.text(msg.course_code);
-      $thisCourseName.text(msg.name)
-    }).fail(function( msg ) {
-      //TODO: reportError(JSON.stringify(msg));
+      $thisCourseName.text(msg.name);
+    }).fail(function(jqXHR, textStatus, errorThrown) {
+      reportError(position,'There was an error changing this course name' + ' (' + jqXHR.status + ' ' + jqXHR.statusText + ')')
```

Comment:
This is just a suggestion as far as readability goes - but I think the code might be more readable if the report error string was written like:

'There was an error changing this course name (' + ...